composer.json

```diff
@@ -15,18 +15,25 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "symfony/symfony": "~2.4",
-    "doctrine/orm": "~2.2,>=2.2.3",
-    "doctrine/doctrine-bundle": "~1.2",
-    "symfony/monolog-bundle": "~2.4"
+    "php": ">=5.6",
+    "symfony/symfony": "~2.6|^3.0",
+    "doctrine/orm": "^2.2.3",
+    "doctrine/doctrine-bundle": "^1.2",
+    "symfony/monolog-bundle": "^2.4"
```

Comment:
Not a fan of this notation, why the change? The difference in the way non-stable and stable versions are treated to me is not intuitive at all. http://blog.madewithlove.be/post/tilde-and-caret-constraints/ (second bullet list, 3rd item)